src/primitive-field/primitive-field.component.ts

```diff
@@ -106,11 +102,15 @@ export class PrimitiveFieldComponent extends AbstractFieldComponent implements O
     this.commitValueChange();
   }
 
-  getTabIndex(): number {
-    return this.tabIndexService.getElemTabIndex(this.path.join('.'));
+  get linkBuilder(): LinkBuilderFunction {
```

Comment:
I am sure that I rebases with master. But if this is part of another PR how I got a conflict with it? I mean it's strange...Anyway I will remove this tomorrow and push again.